### ISSUE TITLE
Splitter: Horizontal

### ISSUE THREAD

**Issue by gle4711** [CodeBeamOrg/CodeBeam.MudExtensions#73]
Hello,

I have tried to nest several. This actually works well too. (https://try.mudblazor.com/snippet/wEmmPPGjSRJDsgKx) What I'm missing is a horizontal splitter so that I can do something similar to this:
https://blazorslidertest.azurewebsites.net  
Could this be implemented?

**Comment by mckaragoz** [comment]
Yeah, actually a horizontal splitter would be great. I also thought to implement this.

But we have a different approach in the splitter, it uses a slider to avoid js and custom drag events (to achieve fastest blazor splitter). And making this horizontal needs lots of CSS arrangement. I didn't have success on my first try.

I also be appreciated if someone try to implement this. Component is ready, we only need to rotate splitter bar and set width and height again.
